lib/webpack.js

```diff
@@ -66,7 +66,8 @@ exports.compile = async (outputPath, benchmarkPath, debug) => {
             }
           }
         ]
-      }
+      },
+      amd: false
```

Comment:
Is there a reason for disabling this?

Reply:
If I remember correctly, `benchmark.js` detects if AMD modules are being used and changes its logic based on that. Webpack is not actually using AMD however; it just creates a false positive. Hence we disable Webpack's AMD handling.

https://github.com/bestiejs/benchmark.js/issues/208